examples/confidential_computing/README.md

```diff
@@ -2,7 +2,9 @@
 
 This is an example of a vm creation with confidential computing,
 encrypted disk using a multiregion (US by default) Cloud HSM key
-and a custom service account with cloud-platform scope.
+and a custom service account with cloud-platform scope. It also
+creates org policies enforcing the use of CMEK encrypted instances
+and confidential computing to all newly created VMs within the project.
```

Comment:
"_...to all newly created VMs within the project..._"

Genuine question: it does not affect the existing VMs that are living on the project?

Reply:
In case that's not affecting existing resource, it may be worth to mention

Reply:
I doesn't affect existing VMs. I added this info to the README file.